Chapter-4-applying-tactical-domain-driven-design/README.adoc

```diff
@@ -7,10 +7,47 @@
+image:https://github.com/evolutionary-architecture/evolutionary-architecture-by-example/actions/workflows/chapter-4-contracts-package-workflow.yml/badge.svg[Build Status]
+image:https://github.com/evolutionary-architecture/evolutionary-architecture-by-example/actions/workflows/chapter-4-contracts-workflow.yml/badge.svg[Build Status]
+image:https://github.com/evolutionary-architecture/evolutionary-architecture-by-example/actions/workflows/chapter-4-package-workflow.yml/badge.svg[Build Status]
+
+== Domain Driven Design in a Nutshell
+=== What is Domain-Driven Design?
+
+Domain-Driven Design (DDD) is a set of techniques and methods that help developers tackle business complexity. It is particularly useful for systems full of business processes and rules that are not easy to understand for people unfamiliar with the business area.
+
+Using DDD is not recommended for trivial business cases, such as when the complexity lies only in the technology itself, or when the application is just CRUD-based and can be designed as simple UI mockups (shallow systems).
+
+=== Domain-Driven Design: Tactical vs. Strategic
+
+Domain-Driven Design techniques are split into two categories: __strategic__ and __tactical__.
+
+==== Strategic
+
+Strategic techniques help you explore, analyse, and understand the problem space. They provide insights into how the company operates, how communication flows, and help build a shared understanding between the development team and the business. Strategic techniques emphasise building a ubiquitous language.
+
+Examples of strategic techniques include:
+
+- link:https://www.strategyzer.com/canvas/business-model-canvas[Business Model Canvas]
+- link:https://learnwardleymapping.com/[Wardley Maps]
+- link:https://www.eventstorming.com/[Event Storming]
+- link:https://domainstorytelling.org/[Domain Storytelling]
+- link:https://vladikk.com/2018/01/26/revisiting-the-basics-of-ddd/[Domain Categorization]
+- link:https://github.com/ddd-crew/context-mapping[Context Mapping]
+
+NOTE: You don't have to use all of them to be successfully. Treat DDD as pharmacy. You should choose the ones that fit your needs and the complexity of the system you are building.
+
+Let's take a look at modelling process.
+
+image::Assets/ddd-starter-modeling-circular.svg[]
+
+Large part of the process is strategic part. That show us how important is to understand the problem space before we start coding. Tactical part is only a small part of the whole process, but still essential to build a good quailty software. 
+
+==== Tactical
+Tactical techniques focus on solution implementation for problems that we already comprehend. They provide a set of patterns and <<building_blocks, building blocks>> that help developers apply the solution (domain model) in the code and maintain it in the long term.
+
+NOTE: Tactical patterns are used only in the _Contracts_ microservice due to its domain complexity.
```

Comment:
There are no tactical patterns, there are patterns of tactical DDD - please replace it as people are crazy about it :)

Reply:
Done :D